connect/mirror/src/test/java/org/apache/kafka/connect/mirror/MirrorSourceConnectorTest.java

```diff
@@ -333,4 +337,136 @@ public String upstreamTopic(String topic) {
                 new CustomReplicationPolicy(), new DefaultTopicFilter(), new DefaultConfigPropertyFilter());
         assertDoesNotThrow(() -> connector.isCycle(".b"));
     }
+
+    @Test
+    public void testTaskConfigTopicPartitions() {
```

Comment:
Seems a little strange to move these tests to the `MirrorSourceConnectorTest` suite since they don't actually test against instances of a `MirrorSourceConnector` (and this might explain why @showuon had trouble finding where these tests were moved to). Did you consider creating new `MirrorSourceConfigTest` and `MirrorCheckpointConfigTest` classes?